.env.sample

```
@@ -8,6 +8,11 @@ DB_NAME=
 DB_USERS_COLLECTION=users
 DB_REFRESH_TOKENS_COLLECTION=refresh_tokens
 DB_FOLLOWERS_COLLECTION=followers
+DB_TWEETS_COLLECTION=tweets
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_TWEETS_COLLECTION key should go before the DB_USERS_COLLECTION key

(UnorderedKey)